pandas/core/indexes/datetimes.py

```diff
@@ -930,6 +930,15 @@ def date_range(
     """
     Return a fixed frequency DatetimeIndex.
 
+    Returns the range of equally spaced timesteps (where the time offset between any
```

Comment:
timesteps -> timestamps?

Reply:
time offset -> difference?

Reply:
I used "timesteps" here to refer to each of the time points in the time series; maybe this is not standard terminology though I think it's used elsewhere in the docs. Maybe should use "time points"?

Reply:
Edited in 9c91ee7, does it look good now?